Game.Checkers/CheckersAction/MoveAction.cs

```diff
@@ -27,6 +28,7 @@ public override CheckersState PerformOn(CheckersState state)
             CheckersState newState = new(state);
             newState.SetPieceAt(Start, Piece.None);
             newState.SetPieceAtWithPossiblePromotion(Finish, movedPiece);
+            newState.CurrentPlayer = state.CurrentPlayer == Player.PlayerOne? Player.PlayerTwo: Player.PlayerOne;
```

Comment:
co to za potwór?

Reply:
W nowym stanie CurrentPlayer to ten drugi Player, czyli jak w starym stanie był PlayerOne to teraz będzie PlayerTwo i na odwrót. Nie wiem jak to lepiej zrobić. Skorzystać z ich wartości i napisać "newState.CurrentPlayer = 3 - state.CurrentPlayer"?